app/src/main/java/com/example/cocktailmaster/data/source/remote/utils/JsonExtension.kt

```diff
@@ -23,10 +25,20 @@ inline fun <reified T> parseToJsonArray(model: String, jsonArray: JSONArray) = j
                 val ingredients: MutableList<String> = ArrayList()
                 val measurements: MutableList<String> = ArrayList()
                 DrinkConstant.INGREDIENTS.forEach {
-                    ingredients.add(jsonObject.optString(it))
+                    val ingredient = jsonObject.optString(it)
+                    if (!TextUtils.equals(ingredient, nullStr) &&
+                        !TextUtils.isEmpty(ingredient)
+                    ) {
+                        ingredients.add(jsonObject.optString(it))
+                    }
                 }
                 DrinkConstant.MEASUREMENTS.forEach {
-                    measurements.add(jsonObject.optString(it))
+                    val measurement = jsonObject.optString(it)
+                    if (!TextUtils.equals(measurement, nullStr) &&
+                        !TextUtils.isEmpty(measurement)
```

Comment:
```suggestion
                        !measurement.isNullOrEmpty
```
 or k cần check null thì dùng isNotEmtye